### PR TITLE
Update dashboard.mdx

### PR DESCRIPTION
**Pull request by omnomagonz**
Updating this doc page to avoid confusion about the availability of this feature/configuration in SaaS and Self-Hosted instances.

### CODE REVIEW

**Comment by omnomagonz** [comment]
@rberrelleza FYI based on our quick chat in Slack.

Would this suffice?

File: versioned_docs/version-1.10/cloud/preview-environments/dashboard.mdx [hunk 1]
```diff
@@ -7,6 +7,9 @@ id: dashboard
 
 import Image from '@theme/Image';
 
+:::caution
+This feature is only available on Okteto Cloud, a product that has been deprecated. These instructions do not apply to, and will not work for, SaaS or Self-Hosted customers. Please us the [GitHub Actions](/cloud/preview-environments/preview-environments-github/) instructions instead to setup Preview Environments in Okteto. This page will soon be deprecated as well.
```
Comment (rberrelleza) [inline versioned_docs/version-1.10/cloud/preview-environments/dashboard.mdx:11]: We haven't made the deprecation official outside of the company. Can we remove that part?

**Review by rberrelleza — APPROVED** [review]

File: versioned_docs/version-1.10/cloud/preview-environments/dashboard.mdx (hunk 1 above)
Comment (omnomagonz) [reply versioned_docs/version-1.10/cloud/preview-environments/dashboard.mdx:11]:
```suggestion
This feature is only available on Okteto Cloud. These instructions do not apply to, and will not work for, SaaS or Self-Hosted customers. Please us the [GitHub Actions](/cloud/preview-environments/preview-environments-github/) instructions instead to setup Preview Environments in Okteto.
```

This work?

Comment (rberrelleza) [reply versioned_docs/version-1.10/cloud/preview-environments/dashboard.mdx:11]: Ship it!

**Review by rberrelleza — APPROVED** [review]

**Pull request merged**